.github/workflows/pilotWrapper.yml

```
@@ -0,0 +1,26 @@
+name: pilot wrapper
+
+on: [push, pull_request]
+
+jobs:
+  PilotWrapper:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: False
+      matrix:
+        python:
+          - 2.6.9
+          - 2.7.5
+          - 2.7.13
+          - 3.6.8
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: prepare environment
+      run: conda create -c conda-forge -n python_${{ matrix.python }} python=${{ matrix.python }}
+    - name: run pilot wrapper test
+      run: |
+        cp tests/Integration/WorkloadManagementSystem/Test_GenerateAndExecutePilotWrapper.py .
+        conda activate python_${{ matrix.python }}
```

Looks like GitHub actions haven't installed `conda` correctly:
```suggestion
        eval "$(conda shell.bash hook)" && conda activate python_${{ matrix.python }}
```

---

I have just done it. Anyway for 2.6 and 2.7.5 there are other issues (see the logs)